conf/nginx.conf.in

```diff
@@ -42,12 +42,6 @@ http {
 	index index.html;
 
 	server {
-	       listen 80 default_server;
-               server_name "~^(?<myhost>.+)$";
-	       rewrite ^ https://$myhost/$request_uri? permanent;
```

Comment:
To @schakrava - school time for Flyer eheh : thinking to yesterday talk about shellinabox addon for Rockstor I immediately thought to required mods to nginx. Are both nginx.conf.in and nginx-prod.conf.in used by Rockstor??? (never had a look inside Rockstor nginx part, so I really don't know)

Thanks :)

Mirko